src/components/domain/BeaconUseSection.tsx

```diff
@@ -18,6 +18,8 @@ export const BeaconUseSection: FunctionComponent<BeaconUseSectionProps> = ({
   changeUri,
   deleteUri,
 }: BeaconUseSectionProps): JSX.Element => {
+  const description = "The main use of this beacon";
```

Comment:
Suggestion to update the description of this text to either:

- Change the use of this beacon
- Change the use of your `sentenceCase(ordinal(index + 1)` beacon